tools/de-identification/google-dp-lib/README.md

```diff
@@ -0,0 +1,13 @@
+# Google Differential Privacy Library
+
+**Primary Focus Area:** De-identification
```

Comment:
I would argue that differential privacy is quite different from de-identification, and hence DP should be its own focus area

Reply:
Totally understand - "de-identification" is a pretty informal term and I agree that some definitions of it exclude differential privacy. For the collab space, we defined it broadly, to mean any technique with the aim of preventing re-identification (and explicitly included differential privacy - see e.g. [here](https://github.com/usnistgov/PrivacyEngCollabSpace/blob/master/tools/README.md)). The goal was to avoid having a whole bunch of separate focus areas (e.g. differential privacy, variants of differential privacy, k-Anonymity, l-diversity, etc), and to drill down on the approach in the keywords (where differential privacy is listed). Does that seem reasonable?

Thanks!

Reply:
Yup sounds reasonable. LGTM